core/src/main/scala/org/broadinstitute/dsp/workbench/welder/StorageLinksAlg.scala

```diff
@@ -0,0 +1,14 @@
+package org.broadinstitute.dsp.workbench.welder
+
+import java.nio.file.Path
+
+import cats.effect.IO
+
+trait StorageLinksAlg {
```

Comment:
seen this a few times but is `alg` short for `algebra`?

Reply:
yea...it is...let me know if you have a better name.